test/tests/imageserver/test_os_image_service.py

```diff
@@ -0,0 +1,411 @@
+        try:
+            if os.path.exists(mountpoint) is False:
+                command = "mkdir " + mountpoint
+                os.popen(command)
+            # use fuseiso tool to mount without root privilege. user need install fuseiso before the test.
+            command = "fuseiso " + file_name + " " + mountpoint
+            os.popen(command)
+            return True
+        except OSError:
+            return False
+
+    def _apply_obmsetting_to_node(self, nodeid):
+        usr = ''
+        pwd = ''
+        response = fit_common.rackhdapi(
+            '/api/2.0/nodes/' + nodeid + '/catalogs/bmc')
+        bmcip = response['json']['data']['IP Address']
+        # Try credential record in config file
+        for creds in fit_common.fitcreds()['bmc']:
+            if fit_common.remote_shell(
+                'ipmitool -I lanplus -H ' + bmcip + ' -U ' +
+                    creds['username'] + ' -P ' + creds['password'] + ' fru')['exitcode'] == 0:
+                usr = creds['username']
+                pwd = creds['password']
+                break
+        # Put the credential to OBM settings
+        if usr != "":
+            payload = {
+                "service": "ipmi-obm-service",
+                "config": {
+                    "host": bmcip,
+                    "user": usr,
+                    "password": pwd},
+                "nodeId": nodeid}
+            api_data = fit_common.rackhdapi("/api/2.0/obms", action='put', payload=payload)
+            if api_data['status'] == 201:
+                return True
+        return False
+
+    def _release(self, file_name, mountpoint):
+        try:
+            # use fusermount to umount the iso loaded by fuseiso without root privilege. User need intall it first.
+            logs.debug("fusermount -u" + mountpoint)
+            os.system("fusermount -u " + mountpoint)
+            logs.debug("rm -d " + mountpoint)
+            os.system("rm -d " + mountpoint)
+            logs.debug("rm " + file_name)
+            os.system("rm " + file_name)
+            return True
+        except OSError:
+            return False
+
+    def _download_file(self, url):
+        logs.debug_3("downloading url= %s" % url)
+        file_name = url.split('/')[-1]
+        if os.path.exists(file_name) is False:
+            u = urllib2.urlopen(url)
+            f = open(file_name, 'wb')
+            meta = u.info()
+            file_size = int(meta.getheaders("Content-Length")[0])
+            logs.debug_3("Downloading: %s Bytes: %s" % (file_name, file_size))
+            file_size_dl = 0
+            block_sz = 8192
+            while True:
+                buffer = u.read(block_sz)
+                if not buffer:
+                    break
+                file_size_dl += len(buffer)
+                f.write(buffer)
+                # logs dose not have ability to draw digital in original place. use print instead.
+                if fit_common.VERBOSITY >= 3:

```

Comment:
Shouldn't need verbosity here.  switch the print statement below to a log message if possible.